__tests__/components/editor/ResourceTemplate.test.js

```diff
@@ -45,9 +45,11 @@ describe('<ResourceTemplate />', () => {
     }
   }
 
+  // Stub `Config.spoofSinopiaServer` static getter to force RT to come from server
+  jest.spyOn(Config, 'spoofSinopiaServer', 'get').mockReturnValue(false)
   const wrapper = shallow(<ResourceTemplate.WrappedComponent resourceTemplateId='resourceTemplate:bf2:Note' />)
   const promise = Promise.resolve(mockResponse(200, null, responseBody))
-  wrapper.instance().getResourceTemplate(promise)
+  wrapper.instance().getResourceTemplatePromise(promise)
```

Comment:
Renamed this method to avoid confusion: `getResourceTemplate` is the name of the main function exported by `sinopiaServer`.